partiql-tests-data/eval/query/select/joins.ion

```diff
@@ -95,3 +95,595 @@ joins::[
     }
   }
 ]
+
```

Comment:
Joins belongs in `query/join/`

Reply:
Yeah, I think that structure makes more sense. Will change the `eval` tests to the correct directory structure.